vm_deploy.sh

```diff
@@ -75,8 +75,23 @@ fi
 echo -e "\n\033[1;32m==== Transferring files to VM ====\033[0m\n"
 rsync -av -e "ssh -o StrictHostKeyChecking=no -i ./id_ed25519" --delete --exclude={'.git','.gitignore','id_ed25519','id_ed25519.pub','cloud-init.yaml','README.md','vm_deploy.sh','vm_destroy.sh','pi_local.sh','commands.txt'} $(pwd) $USER@$(multipass info nameless | grep IPv4 | awk '{ print $2 }'):/home/$USER
 
-echo -e "\n\033[1;32m==== Execute install scripts on VM ====\033[0m\n"
-ssh -o StrictHostKeyChecking=no -i ./id_ed25519 $USER@$(multipass info nameless | grep IPv4 | awk '{ print $2 }') 'cd multicloud_terraform && bash install_terraform.sh && cd aws && bash install_aws.sh && bash deploy_aws.sh'
+# Pass arguments to execute scripts
+if [ "$1" == "aws" ]
+then
+  echo -e "\n\033[1;32m==== Running AWS scripts ====\033[0m\n"
+  ssh -o StrictHostKeyChecking=no -i ./id_ed25519 $USER@$(multipass info nameless | grep IPv4 | awk '{ print $2 }') 'cd multicloud_terraform && bash install_terraform.sh && cd aws && bash install_aws.sh && bash deploy_aws.sh'
```

Comment:
instead of using IF statements, maybe checkout CASE statements. I think you may just want to make a variable called $CLOUD and just reference that like:

`cd multicloud_terraform && bash install_terraform.sh && cd $CLOUD && bash install_$CLOUD.sh && bash deploy_$CLOUD.sh`